src/features/Classes/Class/ClassPage/Actions.jsx

```diff
@@ -74,13 +111,21 @@ const Actions = ({ previousPage }) => {
           alt="menu for actions"
         />
         <Dropdown.Menu>
+          <Dropdown.Item onClick={() => setAssignStaffRole(classLink, classIdDecoded)}>
+            <i className="fa-solid fa-arrow-up-right-from-square mr-2 mb-1" />
+            Class content
+          </Dropdown.Item>
+          <Dropdown.Item onClick={handleManageButton}>
+            <i className="fa-regular fa-chalkboard-user mr-2 mb-1" />
+            Assign instructor
```

Comment:
In the case the class has instructors assigned, the label should be 'Manage instructors'